### PR TITLE
Fix GetLocalHost testutil function for mac users (boot2docker)

### PR DESCRIPTION
**Pull request by sparrc**
Based on google hangout with Marcos Nils

### CODE REVIEW

**Comment by sparrc** [comment]
PS: ignore the circleci job for now, it's not ready and I've disabled for the time being.

**Comment by sparrc** [comment]
@marcosnils @toddboom

File: testutil/testutil.go [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+    "net"
+    "net/url"
+)
 
 var localhost = "localhost"
 
 func GetLocalHost() string {
 	if dockerHostVar := os.Getenv("DOCKER_HOST"); dockerHostVar != "" {
-		return dockerHostVar
+		u, err := url.Parse(dockerHostVar)
```
Comment (marcosnils) [inline testutil/testutil.go]: Needs indentation.

**Comment by marcosnils** [comment]
@sparrc some minor indentation is needed. Besides that LGTM :+1:

**Comment by toddboom** [comment]
@sparrc yep - looks good to me as well! +:100:

**Pull request merged**